packages/lesswrong/lib/notificationTypes.tsx

```diff
@@ -337,6 +348,25 @@ export const NewPublishedDialogueMessagesNotification = registerNotificationType
   causesRedBadge: false,
 });
 
+// New dialogue match between you and another user
+export const NewDialogueMatchNotification = registerNotificationType({
+  name: "newDialogueMatch",
+  userSettingField: "notificationDialogueMatch",
+  async getMessage({documentType, documentId}: GetMessageProps) {
+    const summary = await getDocumentSummary(documentType, documentId)
+    if (summary && summary?.associatedUserName) {
+      return `You matched with ${summary.associatedUserName} for Dialogue Matching!`
+    }
+    return "You have a new Dialogue Match!"
+  },
+  getIcon() {
+    return <DebateIcon style={iconStyles}/>
+  },
+  getLink() {
+    return "/dialogueMatching"
```

Comment:
We may want to change this to the user's page

Reply:
🤷